src/controls.ts

```
@@ -933,13 +938,17 @@ window.addEventListener('keydown', (e) => {
   }
-  if (e.code === 'KeyK' && e.altKey && !e.shiftKey && !e.ctrlKey && !e.metaKey) {
+  if (e.code === 'KeyK' && e.altKey && e.shiftKey && !e.ctrlKey && !e.metaKey) {
     if (sessionStorage.delayLoadUntilFocus) {
       sessionStorage.removeItem('delayLoadUntilFocus')
     } else {
       sessionStorage.setItem('delayLoadUntilFocus', 'true')
     }
   }
+  if (e.code === 'KeyK' && e.altKey && !e.shiftKey && !e.ctrlKey && !e.metaKey) {
+    // eslint-disable-next-line no-debugger
+    debugger
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)